docs/user/dashboard/make-dashboards-interactive.asciidoc

```diff
@@ -174,12 +174,14 @@ To use saved search interactions, open the panel menu, then click *More > View s
 
 Panels have built-in interactive capabilities that apply filters to the dashboard data. For example, when you drag a time range or click a pie slice, a filter for the time range or pie slice is applied. Drilldowns let you customize the interactive behavior while keeping the context of the interaction. 
 
-There are two types of drilldowns you can add to dashboards:
+There are three types of drilldowns you can add to dashboards:
 
-* *Dashboard* &mdash; Navigates you from one dashboard to another dashboard. For example, when can create a drilldown for a *Lens* panel that navigates you from a summary dashboard to a dashboard with a filter for a specific host name.
+* *Dashboard* &mdash; Navigates you from one dashboard to another dashboard. For example, create a drilldown for a *Lens* panel that navigates you from a summary dashboard to a dashboard with a filter for a specific host name.
 
 * *URL* &mdash; Navigates you from a dashboard to an external website. For example, a website with the specific host name as a parameter. 
 
+* *Discover* &mdash; Navigates you from a *Lens* dashboard panel to *Discover*. For example, create a drilldown for a *Lens* visualization that opens the visualization data in *Discover* for further exploration. 
```

Comment:
It also applies the filters based on the interaction (like clicking a pie slice, it will filter down to the documents matching the term the slice represents). Should we add that information here?